src/axom/slic/interface/slic_macros.hpp

```diff
@@ -20,8 +20,12 @@
  * \def SLIC_ERROR( msg )
  * \brief Logs an error and aborts the application.
  * \param [in] msg user-supplied message
- * \note The SLIC_ERROR macro is always active.
  * \warning This macro calls processAbort().
+ * \note The SLIC_ERROR macro is always active.
```

Comment:
Similar comment about grouping methods in Doxygen and using single comment that applies to all.

Reply:
👍  for grouping.

I think it would also be helpful if the doxygen for the collective functions were tagged, e.g. with `\collective`, similar to how we use the `\accelerated` tags. This might make it easier for users to notice this when they are scrolling the header files rather than the generated doxygen.